Backend/wrappers/standardAPI.py

```diff
@@ -55,6 +55,46 @@ def getTweets(self, keyword, since, until, num, location=None):
 
             return tweetsparsed
 
+    def getUsernameFromID(self, id2):
+        user = self.tweepy.get_user(id2)
+        return user._json['screen_name']
+
+    # get list of recent tweets from a user
+    def getTweetsFromUser(self, id2, username, count=20):
+        # get tweets from user ID
+        queryusertweets = []
+        try:
+            if username is not None:
+                queryusertweets = self.tweepy.user_timeline(screen_name=username, count=count)
+            elif id2 > 0:
+                queryusertweets = self.tweepy.user_timeline(id=id2, count=count, page=0)
+            else:
+                return queryusertweets
+        except:
+            print("Can't access user timeline")
+
+        texts = []
+        for status in queryusertweets:
+            texts.append(status._json['text'])
+        return texts
+
+    # get list of userID in a user's follow list
+    def getFollowersID(self, id2, username, count=20):
+        queryfriends_ids = []
+        if username is not None:
+            queryfriends_ids = self.tweepy.friends_ids(screen_name=username)
+        elif id2 > 0:
+            queryfriends_ids = self.tweepy.friends_ids(id=id2)
```

Comment:
Also for later fixing- follower and friend mean very different things on twitter. We'll need to decide which one we want to work with for this, or add the ability to switch between them.

Reply:
Yes its interesting and I realized that halfway in doing this. But yeah i ended up using "following" so basically a user's following list

Reply:
For now, would you mind renaming this, TwitterAPIManager.getFollowersID(), and AlgorithmsManager.append_texts_recursive's local variable followsids to reference friends instead? It seems rather confusing to me to have the function names be so different from what they're doing.